docs/hubs/expense/index.md

```diff
@@ -0,0 +1,30 @@
+---
+heading: Expense APIs
+seo: Expense Hub Overview | Cloud Elements API Docs
+title: Overview
+description: Find API documentation and example API calls.
+layout: sidebarleft
+platform: expense
+breadcrumbs: /docs/hubs/hub-docs.html
+parent: Back to Hub Docs
+order: 1
+sitemap: false
+---
+
+## Expense Hub APIs
+
+Hubs map resources from a collection of endpoints (What we call Elements) into a uniform set of APIs. With Hubs you can integrate an entire category of services (e.g., Event Conferencing, Documents, Finance) through a single set of APIs.
+
+We normalize our API calls for all endpoints, to enable calls between services (e.g. Salesforce to HubSpot). We standardize resources as much as possible across Elements in a Hub, but the Hub APIs are not the least common denominator. For Elements that are more feature rich, that functionality will be explained in the Element specific documentation. We will always provide more features where possible within the Hub. Here are some of the endpoints that are common within Hubs:
+accounts, agents, contacts, documents, files, incidents, leads, opportunities
+
+* Hubs provide uniform APIs to access any collection of Elements or a set of resources.
+* Hubs enable “One-to-Many” access to an entire category of services.
+* Hubs are accessed using a consistent RESTful API with a JSON payload regardless of the technology used at the endpoint.
+* Hubs rapidly translate calls into the semantic and data structure required by each endpoint.
+* Hubs provide a uniform set of interactive API documentation that developers use to access the resources in the Hub.
+* Combine resources from multiple categories providing a consistent set of APIs and documentation to access any collection of resources (e.g., combine Salesforce, Box, and QuickBooks API resources into a Hub).
+
+We’ve categorized the leading cloud applications into Hubs (e.g. Event Conferencing, Documents, and Messaging). You integrate to a Cloud Elements “Hub” via a single RESTful API and your app is instantly connected to all the leading services in that category, no need to do custom and costly integration to each service.
+
+If you need any support integrating our APIs, please let us know. You can [email](mailto:support@cloud-elements.com) or give us a call at +1.866.830.3456. We will do our best to get back to you within 24 hours. Your success is our success.
```

Comment:
We may want to check with cassie before including a phone number, I'm not sure if their team supports a phone line.